### PR TITLE
Updates the README to be a bit cleaner

### PR DESCRIPTION
**Pull request by orowith2os**
I've left some major parts commented out just in case they should be added back.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +68,9 @@ All models should work as long as they are in `.glb` or `.vrm` format. `.gltf` h
    * Release template: `./build-normal-template.sh`
 7. The compiled Godot binary will be available in the `godot/` directory under the `bin/` directory
 
+These instructions are probably more fit for the Godot fork, than here.
```
Comment (you-win) [inline README.md:71]: Commenting this out breaks the readme flow in the "Building from source" section. I'm unwilling to remove this unless the instructions are listed elsewhere.

File: README.md [hunk 2]
```diff
@@ -4,9 +4,11 @@
 
 All models should work as long as they are in `.glb` or `.vrm` format. `.gltf` has not been tested but might work. `.scn` files created with Godot should all import correctly as well.
 
-## Available renderers
+## Available trackers
 * [OpenSeeFace](https://github.com/emilianavt/OpenSeeFace)
 
+*more trackers may be available in the future, such as [MediaPipe](https://google.github.io/mediapipe/)*
```
Comment (you-win) [inline README.md:10]: The "m" in "More" should be capitalized

**Review by you-win — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (orowith2os) [reply README.md:72]: I'll uncomment it for now, could move em over when everything's in more of a working and clean state

**Comment by orowith2os** [comment]
Should be good now

File: README.md [hunk 3]
```diff
@@ -16,21 +18,34 @@ All models should work as long as they are in `.glb` or `.vrm` format. `.gltf` h
...
+### Flatpak
+1. Ensure that [Flathub is enabled on your system](https://flatpak.org/setup/)
+2. Using a graphical interface that supports Flatpak, search for "virtual puppet project"
+   1. Alternatively, you can use `flatpak install vpuppr` to list any apps that have the same string
+   * VPupPr will be available under the name `com.github.virtual_puppet_project.vpuppr`
+   * For the latest (alpha, beta) builds of VPupPr, you can download them from [Flathub Beta](https://beta.flathub.org/)
+4. Hit the install button and run it just like any other app!
+
+
+### (using binaries from the [releases page](https://github.com/virtual-puppet-project/vpuppr/releases))
```
Comment (you-win) [inline README.md:31]: I just noticed that there are extra parentheses around this part, can those be removed. Also please make this a subsection of Linux by making the heading "####"

File: README.md [hunk 4]
```diff
@@ -16,21 +18,34 @@ All models should work as long as they are in `.glb` or `.vrm` format. `.gltf` h
 4. Start the facetracker from within the application
 
 ### Linux
+
+### Flatpak
```
Comment (you-win) [inline README.md:22]: Can this be made a subsection of Linux by making the heading "####"

**Review by you-win — CHANGES_REQUESTED** [review]

**Review by you-win — APPROVED** [review]

**Comment by you-win** [comment]
Is this ready to be merged?

**Comment by orowith2os** [comment]
I think so, after giving it one more read.

**Pull request merged**